### DISCUSSION TITLE
Help with MacOS -> Windows setup

### DISCUSSION THREAD

**Discussion by jeracravo** [apohl79/audiogridder#385] [Help]
Hi there,
I'm trying to get AudioGridder to work on a MacOS 10.14.6 / Windows 10 setup...
The Windows computer would be the server and software is installed fine (apparently)
The thing is that the Mac AudioGridder plugin can't seem to "see" the server at all.
I have my Ethernet adapter set to manual IP4 address (so is the Windows desktop and after turning Firewall off on the Windows system (and opening all sharing configurations I could find), the Mac CAN see the Windows computer in the network... but the Audiogridder plugin still doesn't connect.
Is there any documentation on how to get this to work?
Thanks

**Comment by CaBuffalo** [comment]
When you first time ran AG server, did you grant the firewall access in
Windows 10?

On Mon, Feb 15, 2021 at 12:48 PM jeracravo <notifications@github.com> wrote:

> Hi there,
> I'm trying to get AudioGridder to work on a MacOS 10.14.6 / Windows 10
> setup...
> The Windows computer would be the server and software is installed fine
> (apparently)
> The thing is that the Mac AudioGridder plugin can't seem to "see" the
> server at all.
> I have my Ethernet adapter set to manual IP4 address (so is the Windows
> desktop and after turning Firewall off on the Windows system (and opening
> all sharing configurations I could find), the Mac CAN see the Windows
> computer in the network... but the Audiogridder plugin still doesn't
> connect.
> Is there any documentation on how to get this to work?
> Thanks
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/385>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BCE3MZXSJOGIOHEF3S7GCBZANCNFSM4XVJK4AQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by jeracravo** [reply]
I don't remember having that question pop-up. Let me check...

**Reply by jeracravo** [reply]
Yeah, AudioGridderServer is tagged for both Private and Public networks under the Firewall config (Alllow Apps to communicate...)
Thanks for replying!

**Comment by apohl79** [comment]
Can you ping from mac to win and the other way around? 

Does the ag plugin see the server and you just can't connect to it, or does the server not show up at all?

**Reply by jeracravo** [reply]
Alright... after restarting... the Mac can't see the W10 computer anymore.
Ping doesn't work either way (MacOS > W10 or vice versa)
And when the W10 computer WAS showing up in Finder, I believe the AudioGridder plugin did not see the server (nothing showed up at all
![Screen Shot 2021-02-15 at 4 48 55 PM](https://user-images.githubusercontent.com/12953737/107996352-3048c480-6fae-11eb-897c-89cccb8110bf.png)
![ip](https://user-images.githubusercontent.com/12953737/107996403-4eaec000-6fae-11eb-8450-5d23d68cf544.png)

**Comment by CaBuffalo** [comment]
It looks like your IP is not correct. Are you using DHCP or assigning your
IP address manually?

On Mon, Feb 15, 2021 at 1:53 PM jeracravo <notifications@github.com> wrote:

> Alright... after restarting... the Mac can't see the W10 computer anymore.
> Ping doesn't work either way (MacOS > W10 or vice versa)
> And when the W10 computer WAS showing up in Finder, I believe the
> AudioGridder plugin did not see the server (nothing showed up at all
> [image: Screen Shot 2021-02-15 at 4 48 55 PM]
> <https://user-images.githubusercontent.com/12953737/107996352-3048c480-6fae-11eb-897c-89cccb8110bf.png>
> [image: ip]
> <https://user-images.githubusercontent.com/12953737/107996403-4eaec000-6fae-11eb-8450-5d23d68cf544.png>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/385#discussioncomment-371860>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6C7N6ICL5ZKVW7HBODS7GJWLANCNFSM4XVJK4AQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by jeracravo** [reply]
> It looks like your IP is not correct. Are you using DHCP or assigning your IP address manually?
> -- Kevin Ma Supported by Gmail.

I was using DHCP first... then I switched to manual... should I keep it DHCP and try again?

**Reply by apohl79** [reply]
DHCP requires a DHCP server that assigns IP addresses to client devices. That would usually be the case when you use a router or if you run a DHCP server somewhere on your network.

If you directly connect two computers, you would have to run a DHCP server on one of them, so not sure if that makes sense.

In your screenshots above, your IP configuration is broken. Both computers have to be on the same IP range (subnet). You have to make sure that the subnet mask is identical on both sides, and usually only the last digit of your address differs. 

For example:

On your mac:
IP: 192.168.100.1
Subnet Mask: 255.255.255.0

On windows:
IP: 192.168.100.2
Subnet Mask: 255.255.255.0

Now you should be able to ping the windows IP from the mac and vice versa. If that works, and your firewalls are down, AG should be able to connect.

**Reply by jeracravo** [reply]
I appreciate all the help!
Still doesn't work.
Sometimes the W10 computer shows up on my MacOs Network, it might even let me connect, but still no luck with AG.
Firewall is fully off on W10, I just made it discoverable in all networks (private, public, guest, etc)
Ping doesn't happen either way (Mac>W10 or vice versa)
I wonder WTH am I doing wrong...
Even though Network Setup says it's connected and active on the Mac

![Screen Shot 2021-02-16 at 1 16 27 PM](https://user-images.githubusercontent.com/12953737/108104728-a81ff900-7059-11eb-8b79-29e44e31881d.png)
![Screen Shot 2021-02-16 at 1 27 00 PM](https://user-images.githubusercontent.com/12953737/108105496-b3bfef80-705a-11eb-8080-c814750b9cab.png)

![1](https://user-images.githubusercontent.com/12953737/108105355-8115f700-705a-11eb-8aac-a01789edfd01.png)
![2](https://user-images.githubusercontent.com/12953737/108105385-8d01b900-705a-11eb-8993-bf4b3a876526.png)
![3](https://user-images.githubusercontent.com/12953737/108105394-8f641300-705a-11eb-90ce-bb4ec25e56f6.png)

**Comment by CaBuffalo** [comment]
Can you use DHCP on both systems?

**Reply by jeracravo** [reply]
As Andreas mentioned, I'm not sure I should use DHCP

**Comment by CaBuffalo** [comment]
DHCP works fine to me. Give it a try if you have a router.

On Tue, Feb 16, 2021 at 10:59 AM jeracravo <notifications@github.com> wrote:

> As Andreas mentioned, I'm not sure I should use DHCP
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/385#discussioncomment-374686>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6AAB5HQO7YVFNLJM2LS7K6CXANCNFSM4XVJK4AQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by jeracravo** [reply]
I don't have a router available. The machines are connected directly to each other.

**Comment by CaBuffalo** [comment]
I see. Ok, you will need a crossover cable to connect them directly. For IP
addresses, you can use what Andreas mentioned above.

On Tue, Feb 16, 2021 at 11:46 AM jeracravo <notifications@github.com> wrote:

> I don't have a router available. The machines are connected directly to
> each other.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/385#discussioncomment-374826>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6FHG3ZLG2EKD7NVFKTS7LDQHANCNFSM4XVJK4AQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by jeracravo** [reply]
What a weird situation...
So I finally got a crossover cable, and now they don't even see each other in the network.
Should I borrow a router and try with it? It really doesn't make any sense to me.

**Comment by CaBuffalo** [comment]
Yes, if you can use a router. It makes everything much easier for you.

Remember, you can't use crossover cables with a router.

On Thu, Feb 25, 2021 at 10:25 AM jeracravo <notifications@github.com> wrote:

> What a weird situation...
> So I finally got a crossover cable, and now they don't even see each other
> in the network.
> Should I borrow a router and try with it? It really doesn't make any sense
> to me.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/385#discussioncomment-406808>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BEYSHMOJKUODWYKMTTA2I3BANCNFSM4XVJK4AQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.
